cmd/cluster/etcd_health.go

```diff
@@ -51,38 +51,47 @@ func (capture *LogCapture) Write(p []byte) (n int, err error) {
 }
 
 type etcdHealthCheckOptions struct {
-	reason string
+	reason    string
+	clusterID string
 }
 
 func newCmdEtcdHealthCheck() *cobra.Command {
 	opts := etcdHealthCheckOptions{}
 	cmd := &cobra.Command{
-		Use:               "etcd-health-check <cluster-id> --reason <reason for escalation>",
+		Use:               "etcd-health-check --cluster-id <cluster-id> --reason <reason for escalation>",
 		Short:             "Checks the etcd components and member health",
 		Long:              `Checks etcd component health status for member replacement`,
-		Args:              cobra.ExactArgs(1),
+		Args:              cobra.NoArgs,
 		DisableAutoGenTag: true,
 		Run: func(cmd *cobra.Command, args []string) {
-			cmdutil.CheckErr(EtcdHealthCheck(args[0], opts))
+			cmdutil.CheckErr(EtcdHealthCheck(opts))
 		},
 	}
 
+	cmd.Flags().StringVar(&opts.clusterID, "cluster-id", "", "Provide the internal Cluster ID or name to perform health check on")
 	cmd.Flags().StringVar(&opts.reason, "reason", "", "Specify a reason for privilege escalation")
-	err := cmd.MarkFlagRequired("reason")
+
+	err := cmd.MarkFlagRequired("cluster-id")
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "WARNING: failed to mark flag '--cluster-id' as required: %v", err)
+	}
```

Comment:
You aren't checking this error elsewhere. I usually _don't_ check this, it only fails if you haven't created a flag that matches. Either way, let's be consistent.